packages/graphql/src/classes/Neo4jGraphQL.ts

```diff
@@ -64,14 +64,15 @@ class Neo4jGraphQL {
 
     private _nodes?: Node[];
     private _relationships?: Relationship[];
-
+    private _plugins?: Neo4jGraphQLPlugins;
```

Comment:
**suggestion:** should we only use underscores where necessary for class members? I only used it on `_nodes` and `_relationships` as they needed getters (can't have member and getter with same name).

```suggestion
    private plugins?: Neo4jGraphQLPlugins;
```

Reply:
Good catch!